### DISCUSSION TITLE
Mandatory lockscreen setting for institutional google account

### DISCUSSION THREAD

**Discussion by wavelov3r** [microsoft/WSA#449] [General]
Hello, i have an istitutional google account. When i try to add as an Android account, it tells me that the organization require a mandatory lockscreen setting.
I've tried to set it (the only available is swipe), but it's not sufficient.

Does have someone some ideas to figure out?

Edit: i am unable to use WSA anymore. maybe it waits for a swipe to unlock the screen....?

**Comment by HotCakeX** [comment]
I need a lock screen such as PIN too.
I want to use Microsoft Authenticator app on WSA but its autofill feature doesn't work and I can't access my passwords in Microsoft Authenticator without having a screen lock.

**Comment by wavelov3r** [comment]
I think something can be done rooting it.. Like a fake lockscreen

**Reply by HotCakeX** [reply]
I don't want to introduce 3rd party stuff that I don't trust to the WSA.
It'd be so much better if Microsoft provided this feature natively.
